autopush/tests/test_db.py

```diff
@@ -301,106 +313,45 @@ def test_message_storage(self):
-        ttl = int(time.time())+100
         time1, time2 = self._nstime(), self._nstime()+1
-        message.store_message(self.uaid, chid, time1, ttl, data1, {})
-        message.store_message(self.uaid, chid, time1, ttl, data2, {})
-        message.store_message(self.uaid, chid2, time2, ttl, data1, {})
+        message.store_message(notif1, timestamp=time1)
+        message.store_message(notif2, timestamp=time1)
+        message.store_message(notif3, timestamp=time2)
 
-        all_messages = list(message.fetch_messages(self.uaid))
+        all_messages = list(message.fetch_messages(uuid.UUID(self.uaid)))
         eq_(len(all_messages), 2)
 
-        message.delete_messages_for_channel(self.uaid, chid2)
-        all_messages = list(message.fetch_messages(self.uaid))
-        eq_(len(all_messages), 1)
-
-        message.delete_message(self.uaid, chid, time1)
-        all_messages = list(message.fetch_messages(self.uaid))
-        eq_(len(all_messages), 0)
-
-    def test_delete_user(self):
-        chid = str(uuid.uuid4())
-        chid2 = str(uuid.uuid4())
-        m = get_rotating_message_table()
-        message = Message(m, SinkMetrics())
-        message.register_channel(self.uaid, chid)
-        message.register_channel(self.uaid, chid2)
-
-        data1 = str(uuid.uuid4())
-        data2 = str(uuid.uuid4())
-        ttl = int(time.time())+100
-        time1, time2, time3 = self._nstime(), self._nstime(), self._nstime()+1
-        message.store_message(self.uaid, chid, time1, ttl, data1, {})
-        message.store_message(self.uaid, chid2, time2, ttl, data2, {})
-        message.store_message(self.uaid, chid2, time3, ttl, data1, {})
-
-        message.delete_user(self.uaid)
-        all_messages = list(message.fetch_messages(self.uaid))
-        eq_(len(all_messages), 0)
-
-    def test_message_delete_pagination(self):
-        def make_messages(channel_id, count, rtable):
-            t = self._nstime()
-            ttl = int(time.time())+200
-            for i in range(count):
-                rtable.append(
-                    (self.uaid, channel_id, str(uuid.uuid4()), ttl, {}, t+i)
-                )
-            return rtable
-
-        chid = str(uuid.uuid4())
-        m = get_rotating_message_table()
-        message = Message(m, SinkMetrics())
-        message.register_channel(self.uaid, chid)
-
-        # Shove 80 messages in
-        m = []
-        for message_args in make_messages(chid, 80, m):
-            message.store_message(*message_args)
-
-        # Verify we can see them all
-        all_messages = list(message.fetch_messages(self.uaid, limit=100))
-        eq_(len(all_messages), 80)
-
-        # Delete them all
-        message.delete_messages_for_channel(self.uaid, chid)
-
-        # Verify they're gone
-        all_messages = list(message.fetch_messages(self.uaid, limit=100))
-        eq_(len(all_messages), 0)
-
     def test_message_delete_fail_condition(self):
+        notif = WebPushNotification(
```

Comment:
Why not call `make_webpush_notification` and add the extra fields? (or since they're None, adding them to the function?)